CustomAugments.cpp

```diff
@@ -487,6 +487,24 @@ HOOK_METHOD_PRIORITY(ShipObject, GetAugmentationValue, 1000, (const std::string&
     return ret;
 }
 
+HOOK_METHOD(ShipObject, GetAugmentationValue, (const std::string& name) -> float)
+{
+    LOG_HOOK("HOOK_METHOD -> ShipObject::GetAugmentationValue -> Begin (CustomAugments.cpp)\n")
+    float ret = super(name);
+
+    auto context = G_->getLuaContext();
+    SWIG_NewPointerObj(context->GetLua(), G_->GetShipManager(this->iShipId), context->getLibScript()->types.pShipManager, 0);
```

Comment:
Do we want to have the ShipManager be the argument, or would just the ship ID or the ShipInfo be better?

Reply:
ShipManager is better since its consistent with other callbacks